ietf/meeting/utils.py

```diff
@@ -155,6 +156,8 @@ def finalize(meeting):
             else:
                 sp.rev = '00'
             sp.save()
+        if meeting.number >= '118':
```

Comment:
Duh, that was a last-minute addition, and it was late. Up to a minute before I committed, `generate_bluesheet` was unconditional. The original ticket said to start giving the live view of attendance with 118, but we later changed that to any un-finalized meeting. The intent of this check was to avoid overwriting good pre-existing bluesheet files in the unlikely event that, say, we had to un-finalize and re-finalize 107 (before we had `Attended` records). But that's pretty far-fetched, and maybe I should just remove that check.

Reply:
I agree it's unlikely, but I like the idea of a safeguard.

Reply:
Nah, keep the check, but roll it back to 108 (the earliest `Attended` records).